File: PHP/Shirt.php
<?php
require_once 'Clothing.php';

class Shirt extends Clothing {
    private $color;
    private $sleeveType;

    public function __construct($idProduct, $name, $brand, $price, $size, $material, $gender, $color, $sleeveType) {
        parent::__construct($idProduct, $name, $brand, $price, $size, $material, $gender);
        $this->color = $color;
        $this->sleeveType = $sleeveType;
    }

    public function getColor() {
        return $this->color;
    }

    public function setColor($color) {
        $this->color = $color;
    }

    public function getSleeveType() {
        return $this->sleeveType;
    }

    public function setSleeveType($sleeveType) {
        $this->sleeveType = $sleeveType;
    }
}
?>

File: PYTHON/shirt.py
from clothing import Clothing

class Shirt(Clothing):
    def __init__(self, idProduct, name, brand, price, size, material, gender, color, sleeveType):
        super().__init__(idProduct, name, brand, price, size, material, gender)
        self.color = color
        self.sleeveType = sleeveType

    def getColor(self):
        return self.color

    def setColor(self, color):
        self.color = color

    def getSleeveType(self):
        return self.sleeveType

    def setSleeveType(self, sleeveType):
        self.sleeveType = sleeveType
